memento.root/usr/share/clearwater/cassandra-schemas/memento.sh

```diff
@@ -1,8 +1,119 @@
-#! /bin/bash
-if [[ ! -e /var/lib/cassandra/data/memento ]];
-then
-    echo "CREATE KEYSPACE memento WITH strategy_class='org.apache.cassandra.locator.SimpleStrategy' AND strategy_options:replication_factor=2;
-          USE memento;
-          CREATE TABLE call_lists (impu text PRIMARY KEY, dummy text) WITH read_repair_chance = 1.0;
-    " | cqlsh -2
+#!/bin/bash
+
+keyspace=$(basename $0|sed -e 's#^\(.*\)[.]sh$#\1#')
+
+. /etc/clearwater/config
+if [ ! -z $signaling_namespace ]; then
+    if [ $EUID -ne 0 ]; then
+        echo "When using multiple networks, schema creation must be run as root"
+        exit 2
+    fi
+    namespace_prefix="ip netns exec $signaling_namespace"
+fi
+
+if [[ ! -e /var/lib/cassandra/data/${keyspace} ]]; then
+    header="Waiting for Cassandra"
+    let "cnt=0"
```

Comment:
Yes, but I am printing out a different progress character depending on what port I'm waiting to come alive.